README.md

```diff
@@ -1,52 +1,51 @@
-<p align="center">
-  <a href="https://impulso.network/" target="_blank" alt="Impulso Network">
-    <img src="https://impulso.network/assets/images/impulsonetwork-logo.svg" alt="Impulso" width="350px" />
-  </a>
-</p>
+<img src="https://impulso.network/assets/images/impulsonetwork-logo.svg" style="width: 350px">
 
+[![Build Status](https://dev.azure.com/impulsonetwork/Atena/_apis/build/status/impulsonetwork.atena)](https://dev.azure.com/impulsonetwork/Atena/_build/latest?definitionId=3)
 [![Website](https://img.shields.io/website-up-down-green-red/http/shields.io.svg?label=about)](http://impulso.network)
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](LICENSE)
 [![Twitter Follow](https://img.shields.io/twitter/follow/impulsonetwork.svg?style=social&label=Follow)](https://twitter.com/impulsonetwork)
 
+
 ## Atena v.0.1
 
-Atena is an initiative of the impulse in collaboration with several Impulsers, with the objective to promote the engagement and to reward the activities and efforts of each person in the Impulse. In it you will be able to acquire Experience Points (XP) through the execution of several activities and based on these two factors the Impulsers will receive levels and achievements recognizing the effort expended in the activities.
+A Atena é uma iniciativa da Impulso Network em colaboração com vários Impulsers, com o objetivo de promover o engajamento e recompensar as atividades e esforços de cada pessoa na Impulso. Nele você poderá adquirir Pontos de Experiência (XP) através da execução de diversas atividades e com base nesses dois fatores os Impulsers receberão níveis e conquistas, reconhecendo o esforço despendido nas atividades.
 
-## Setup
+## Configuração
 
-A step-by-step  of the minimal setup you need to get a Atena running.
+Um passo-a-passo da configuração mínima que você precisa para obter o The Arena em execução.
```

Comment:
The Arena?

Reply:
Deveria ser "Um passo-a-passo da configuração mínima que você precisa para obter o Atena em execução."